src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -2,7 +2,7 @@ import Vue from 'vue'
 import FtIconButton from '../ft-icon-button/ft-icon-button.vue'
 import { mapActions } from 'vuex'
 import i18n from '../../i18n/index'
-import { copyToClipboard, openExternalLink, showToast } from '../../helpers/utils'
+import { copyToClipboard, openExternalLink, showToast, toLocalePublicationString } from '../../helpers/utils'
```

Comment:
This is getting a bit long... (109 chars)
```suggestion
import { 
  copyToClipboard, 
  openExternalLink, 
  showToast, 
  toLocalePublicationString
} from '../../helpers/utils'
```